docs/nachet-inference-documentation.md

```diff
@@ -41,20 +43,21 @@ end
 ```
 
 ### Models
+
 A model is an AI model that is a part of a pipeline. A model accepts images as
 input and returns JSON as output. Generally, this JSON contains the coordinates
 of objects in the source image, that the model may pass along to feed the next
 step of the pipeline.
 
-
 ### Model from Frontend
+
 On the frontend interface, a pipeline will be called a model, because the user
 will not be aware of the difference. From the user's perspective, they send data
 to a model and receive the result.
 
 *Suggestion: we could call the pipeline a method if we don't want to mix terms.*
 
-# Sequence Diagram for inference request 1.0.0
+## Sequence Diagram for inference request 1.2.1
 
 ```mermaid
 sequenceDiagram
```

Comment:
add title: property to your diagrams (with the versioning that you gave above)

Reply:
?

Reply:
I add the title on mermaid, but it is really tiny and hard to see. I try to increase the title font, but the sequence diagram doesn't have a attribute title to style.

Reply:
we can live with that for now